crates/cairo-addons/src/vm/memory_segments.rs

```diff
@@ -10,8 +10,26 @@ pub struct PyMemorySegmentManager {
     pub(crate) runner: *mut RustCairoRunner,
 }
 
+/// Enables syntax `segments.memory.<op>`
+#[pyclass(name = "MemoryWrapper", unsendable)]
+pub struct PyMemoryWrapper {
+    pub(crate) runner: *mut RustCairoRunner,
+}
+
+#[pymethods]
+impl PyMemoryWrapper {
+    fn get(&self, key: PyRelocatable) -> Option<PyMaybeRelocatable> {
+        unsafe { (*self.runner).vm.get_maybe(&key.inner).map(PyMaybeRelocatable::from) }
```

Comment:
why unsafe?

Reply:
that's how it was. I'm not modifying this